src/plugins/intel_cpu/src/config.cpp

```diff
@@ -89,9 +89,19 @@ void Config::readProperties(const ov::AnyMap& prop, const ModelType modelType) {
                                    ". Expected only ov::Affinity::CORE/NUMA/HYBRID_AWARE.");
                 }
             }
+            if (key == ov::num_streams.name()) {
+                ov::Any value = val.as<std::string>();
+                auto streams_value = value.as<ov::streams::Num>();
+                if (streams_value == ov::streams::NUMA) {
+                    latencyThreadingMode = Config::LatencyThreadingMode::PER_NUMA_NODE;
+                } else if (streams_value == ov::streams::AUTO) {
+                    hintPerfMode = ov::hint::PerformanceMode::THROUGHPUT;
+                    changedHintPerfMode = true;
+                }
```

Comment:
Do we need check the invalid value, such as `streams_value<-2 `?

Reply:
Because the validation is checked in ahead `streamExecutorConfig.set_property()`, so I think no need to check again here.